src/app/core/entity/default-datatype/view.directive.ts

```diff
@@ -12,6 +12,12 @@ export abstract class ViewDirective<T, C = any> implements OnChanges {
   /** indicating that the value is not in its original state, so that components can explain this to the user */
   isPartiallyAnonymized: boolean;
 
+  /**
+   * Attention:
+   * When content is loaded async in your child component, you need to manually trigger the change detection
+   * See: https://angularindepth.com/posts/1054/here-is-what-you-need-to-know-about-dynamic-components-in-angular#ngonchanges
+   *
+   */
```

Comment:
After reading through this blogpost  I had another idea how me might be able to get rid of this issue in our dynamic components:
We could also `await` the `ngOnChanges` call [where we create the dynamic component](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/config/dynamic-components/dynamic-component.directive.ts#L94).
This would make the `markForCheck` call unnecessary and it also mirrors the "normal" behaviour when using the component in a template and passing the entity as a input parameter.